src/js/engagement_view/src/tests/engagementView/data/graphVizData.tsx

```diff
@@ -1,76 +1,85 @@
 export const vizGraphData = [
 	{
-		uid: 10005,
-		node_key: "3db5c6b8-1c4e-414f-9947-7a02f07156fd",
-		dgraph_type: ["Process"],
-		process_name: "cmd.exe",
-		process_id: 5824,
-		children: [
-			{
-				uid: 10032,
-				node_key: "69c90caa-cbc6-4b80-8f29-60f501aeb114",
-				dgraph_type: ["Process"],
-				process_name: "svchost.exe",
-				process_id: 6132,
-			},
-		],
-		risks: [
-			{
-				uid: 10133,
-				dgraph_type: ["Risk"],
-				node_key: "Suspicious svchost",
-				analyzer_name: "Suspicious svchost",
-				risk_score: 75,
-			},
-			{
-				uid: 10138,
-				dgraph_type: ["Risk"],
-				node_key: "Rare Parent of cmd.exe",
-				analyzer_name: "Rare Parent of cmd.exe",
-				risk_score: 10,
-			},
-		],
-	},
-	{
-		uid: 10013,
-		node_key: "DESKTOP-FVSHABR",
+		uid: 40,
+		node_key: "53e2abc7-ce21-4be0-b70a-4c835eefecd1",
 		dgraph_type: ["Asset"],
+		display: "Asset",
 		hostname: "DESKTOP-FVSHABR",
 		asset_ip: null,
 		asset_processes: [
 			{
-				uid: 10005,
-				node_key: "3db5c6b8-1c4e-414f-9947-7a02f07156fd",
+				uid: 171,
+				node_key: "9388ceda-91fc-49c8-989c-8968ad93d0e8",
 				dgraph_type: ["Process"],
+				display: null,
+				process_name: "dropper.exe",
+				process_id: 4164,
+			},
+			{
+				uid: 193,
+				node_key: "a9ca278c-0f52-4e89-821c-876d750ad1dd",
+				dgraph_type: ["Process"],
+				display: null,
 				process_name: "cmd.exe",
 				process_id: 5824,
 			},
 			{
-				uid: 10030,
-				node_key: "d7f7d377-1c7c-486f-9956-6ef996c5b606",
+				uid: 201,
+				node_key: "a0c5dacc-fd53-450f-bb99-207467b9c7ee",
 				dgraph_type: ["Process"],
-				process_name: "dropper.exe",
-				process_id: 4164,
+				display: null,
+				process_name: "cmd.exe",
+				process_id: 5824,
 			},
 			{
-				uid: 10032,
-				node_key: "69c90caa-cbc6-4b80-8f29-60f501aeb114",
+				uid: 207,
+				node_key: "739c609a-05ac-4bed-949c-387efb06114a",
 				dgraph_type: ["Process"],
+				display: null,
 				process_name: "svchost.exe",
 				process_id: 6132,
 			},
 		],
 		files_on_asset: null,
 		risks: [
 			{
-				uid: 10133,
+				uid: 213,
+				dgraph_type: ["Risk"],
+				display: null,
+				node_key: "Rare Parent of cmd.exe",
+				analyzer_name: "Rare Parent of cmd.exe",
+				risk_score: 10,
+			},
+			{
+				uid: 220,
 				dgraph_type: ["Risk"],
+				display: null,
 				node_key: "Suspicious svchost",
 				analyzer_name: "Suspicious svchost",
 				risk_score: 75,
 			},
+		],
+	},
+	{
+		uid: 171,
+		node_key: "9388ceda-91fc-49c8-989c-8968ad93d0e8",
+		dgraph_type: ["Process"],
+		process_name: "dropper.exe",
+		process_id: 4164,
+		display: "Process",
```

Comment:
will dgraph_type and display ever not match?

Reply:
They may, we default to dgraph_type but it can be overridden. :)